tools/cockpit.spec

```diff
@@ -459,6 +459,7 @@ Suggests: python3
 
 # prevent hard python3 dependency for cockpit-desktop, it falls back to other browsers
 %global __requires_exclude_from ^%{_libexecdir}/cockpit-client$
+%global __requires_exclude_from ^%{_libexecdir}/cockpit-client-ssh$
```

Comment:
You can only set this once, so the original regexp needs to be extended instead. I suggest to do `client.*$` -- while it's equivalent to just `client`, it makes it explicit that we expect suffixes there.

Reply:
I just conditionalised the install to dodge the problem.